wizard_builder/tests/test_sites.py

```diff
@@ -1,10 +1,11 @@
 from mock import patch
-from wizard_builder.models import PageBase, QuestionPage, SingleLineText
 
 from django.conf import settings
 from django.contrib.sites.models import Site
 from django.test import TestCase, override_settings
 
+from ..models import PageBase, QuestionPage, SingleLineText
```

Comment:
is this just aesthetic preference? (i'm fine with it just trying to get better at python)